### PR TITLE
Ask odoo version first

### PR DESCRIPTION
**Pull request by yelizariev**
It allows to quickly apply default values on all other options by pressing Enter button.
Such approach is especially useful for local modules development

----------------
I'd actually like to move just odoo_version attribute, but the attributes are grouped, so I moved whole ``odoo_`` group and reordered making first the attributes most important for modules development.

As for now, I need to slowly click enter to don't miss odoo_version question -- the only question where I don't use default value

------------------

I just noticed that it's stable branch. Am I supposed to send it to devel branch instead?

### CODE REVIEW

File: copier.yml [hunk 1]
```diff
@@ -77,6 +77,75 @@ _migrations:
...
+    Examples: en_US, es_ES, es_VE...
+
+odoo_admin_password:
+  secret: true
+  default: example-admin-password
+  type: str
+  help: >-
+    💡 To auto-generate strong passwords, see https://ddg.gg/?q=password+64+strong
+
+    ⚠️ This password is critical for security, especially if you have set odoo_listdb to
```
Comment (joao-p-marques) [inline copier.yml:128]: As we are changing the order, maybe change this to 
```suggestion
    ⚠️ This password is critical for security, especially if you set odoo_listdb to
```
or put `odoo_listdb` before this.

**Review by joao-p-marques — COMMENTED** [review]
LGTM :+1: 
Just a small comment

**Review by Yajo — APPROVED** [review]

File: copier.yml (hunk 1 above)
Comment (yelizariev) [reply copier.yml]: Yeah, makes sense

**Review by joao-p-marques — APPROVED** [review]
LGTM :+1:

**Pull request merged**